x-pack/plugins/security/server/saved_objects/saved_objects_security_extension.ts

```diff
@@ -11,7 +11,7 @@ import type {
   SavedObjectsResolveResponse,
 } from '@kbn/core-saved-objects-api-server';
 import type { SavedObjectsClient } from '@kbn/core-saved-objects-api-server-internal';
-import { isBulkResolveError } from '@kbn/core-saved-objects-api-server-internal/src/lib/internal_bulk_resolve';
+import { isBulkResolveError } from '@kbn/core-saved-objects-api-server-internal/src/lib/apis/internals/internal_bulk_resolve';
```

Comment:
@elastic/kibana-security 🙈 nested import for internal core package.

Reply:
🙈